test/Microsoft.AspNetCore.Mvc.Core.Test/Authorization/AuthorizeFilterTest.cs

```diff
@@ -503,7 +503,7 @@ public void CreateInstance_ReturnsNewFilterIfPolicyAndPolicyProviderAreNotSet(Au
             httpContext.SetupGet(c => c.RequestServices).Returns(serviceProvider);
             auth.Setup(c => c.AuthenticateAsync("Bearer")).ReturnsAsync(bearerPrincipal);
             auth.Setup(c => c.AuthenticateAsync("Basic")).ReturnsAsync(basicPrincipal);
-            auth.Setup(c => c.AuthenticateAsync("Fails")).ReturnsAsync(null);
+            auth.Setup(c => c.AuthenticateAsync("Fails")).ReturnsAsync(default(ClaimsPrincipal));
```

Comment:
This will be required for Moq 4.7.1 to disambiguate the call between `ReturnsAsync(T)` and a new overload `ReturnsAsync(Func<T>)`